gradle.properties

```diff
@@ -1,4 +1,4 @@
 group = com.enonic.lib
 projectName = lib-cron
 xpVersion=7.0.0
-version=1.0.0
+version=1.3
```

Comment:
Should not we follow X.Y.Z pattern? It should also be -SNAPSHOT

Reply:
It was commited by mistake, should have been in other PR.